tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -200,6 +203,30 @@ export class PodmanDesktopRunner {
       console.log(`Video file saved as: ${videoPath}`);
     }
     this._running = false;
+    await this.removeTracesOnFinished();
+  }
+
+  async removeTracesOnFinished(): Promise<void> {
+    const rawTracesPath = join(this._testOutput, 'traces', 'raw');
+
+    if (existsSync(rawTracesPath)) {
+      console.log(`Removing raw traces folder: ${rawTracesPath}`);
+      rmSync(rawTracesPath, { recursive: true, force: true, maxRetries: 5 });
+    }
+
+    if (this._testStatus && !process.env.KEEP_TRACES_ON_PASS && this._videoAndTraceName) {
+      const tracesPath = join(this._testOutput, 'traces', `${this._videoAndTraceName}_trace.zip`);
+      if (existsSync(tracesPath)) {
+        console.log(`Removing traces folder: ${tracesPath}`);
+        rmSync(tracesPath, { recursive: true, force: true, maxRetries: 5 });
+      }
+
+      const videoPath = join(this._testOutput, 'videos', `${this._videoAndTraceName}.webm`);
+      if (existsSync(videoPath)) {
+        console.log(`Removing video folder: ${videoPath}`);
+        rmSync(videoPath, { recursive: true, force: true, maxRetries: 5 });
```

Comment:
In my opinion it's still useful to have the video recording even if the test passes, to be able to rule out false negatives and general debugging

Reply:
a `KEEP_VIDEOS_ON_PASS` env var ?

Reply:
I mean I can change it to use a different env var for video if you want but I thought using two different ones seems like a lot and @odockal wanted to have the videos removed by default also in order to have the final trace files as small as possible.

@odockal what do you think?

Reply:
I would not mind I guess. Also, from my point of view I do not recall opening videos or even downloading artifacts from passed tests. On the other hand, when you develop new test, it might be useful. So let's have more fined grained options. It is not really causing any harm. but I would not keep it as default.

Reply:
Example for the smoke run: Videos are 20 MB, traces (just zips) are 90 MB. so it is roughly 1:4,5 rate videos x traces. I will leave the decision on you, guys. If @ScrewTSW wants it since he is using it, I would not mind.

Reply:
Added a new env var named KEEP_VIDEOS_ON_PASS to enable video saving separate from traces.